### PR TITLE
[fr] Another batch of French translations

### PR DESCRIPTION
**Pull request by HavokInspiration**
Follows #4787, #4784, #4790, #4794, #4795, #4796, #4803, #4807, #4810, #4812, #4821, #4824, #4826, #4822 

There are some modifications on the english versions as well.

### CODE REVIEW

File: fr/development/routing.rst [hunk 1]
```diff
@@ -712,24 +712,39 @@ Routing des Extensions de Fichier
...
+
 Pour manipuler différentes extensions de fichier avec vos routes, vous pouvez
-ajouter ce qui suit dans votre fichier de config des routes::
+définir vos extensions de manière globale ou dans un *scope*. Définir des
+extensions globales peut se faire via la méthode static :php:meth:`Router::extensions()`
+
+    Router::extensions(['json', 'xml']);
+    // ...
+
+Ceci affectera **toutes** les routes qui seront connectés **après** cet appel,
```
Comment (antograssiot) [inline fr/development/routing.rst]: connectées ?

File: fr/development/routing.rst [hunk 2]
```diff
@@ -712,24 +712,39 @@ Routing des Extensions de Fichier
...
+        // ...
     });
 
-Cela activera les extensions de nom pour toutes les routes déclarées **après**
-l'appel de cette méthode. Par défaut, les extensions que vous avez déclarées
-seront fusionnées avec la liste des extensions existantes.
+Cela activera les extensions pour toutes les routes qui seront définies dans ce
+scope **après** l'appel à ``extensions()``, tout en incluant les routes inclus
+dans les scopes imbriqués. De la même manière que la méthode :php:meth:`Router::extensions()`,
+toutes les routes connectés avant cet appel n'hériteront pas de ces extensions.
```
Comment (antograssiot) [inline fr/development/routing.rst]: connectées

File: fr/development/routing.rst [hunk 3]
```diff
@@ -712,24 +712,39 @@ Routing des Extensions de Fichier
...
 
 .. note::
 
     Le réglage des extensions devrait être la première chose que vous devriez
     faire dans un scope, car les extensions seront appliquées uniquement aux
     routes qui sont définies **après** la déclaration des extensions.
 
+    Lorsque vous définissez des routes dans le même scope mais dans deux appels
+    différents, les extensions ne seront pas hérités d'un appel à l'autre.
```
Comment (antograssiot) [inline fr/development/routing.rst]: héritées

**Comment by HavokInspiration** [comment]
@antograssiot I fixed the grammar. I'll take a look at the build issue.

**Comment by antograssiot** [comment]
Merci @HavokInspiration

**Pull request merged**